src/components/auth/AuthStatus.tsx

```
@@ -0,0 +1,85 @@
+"use client";
+
+import { useState } from "react";
+import { useSession, signOut } from "next-auth/react";
+import Link from "next/link";
+import { ChevronDownIcon, UserIcon } from "@heroicons/react/24/outline";
+import Button from "../ui/Button";
```

ESLint

[error] 'Button' is defined but never used.

(@typescript-eslint/no-unused-vars)